frontend/src/components/Board/Board.jsx

```diff
@@ -0,0 +1,42 @@
+import React, {useState} from 'react';
+import {
+  BoardInfoContainer,
+  BoardInfo,
+  BoardImage,
+  BoardTitle,
+  AuthorInfoContainer,
+  AuthorName,
+  AuthorAvatar
+} from "./Board.styled";
+import DotsButton from "../DotsButton/DotsButton";
+import Dropdown from '../Dropdown/Dropdown';
+import { FaUserCircle } from 'react-icons/fa';
+
+const Board = () => {
+  const [clickHandler, setClickHandler] = useState(false)
+
+  return (
+      <BoardInfoContainer>
+        <BoardInfo>
+          <BoardImage />
+          <BoardTitle onClick={() => setClickHandler(!clickHandler)}>
+            nazwa_tablicy
+          </BoardTitle>
+          <DotsButton>
+            <Dropdown>
+            </Dropdown>
+          </DotsButton>
+        </BoardInfo>
+        <AuthorInfoContainer>
+          <AuthorName>
+            nazwa_uzytkownika
+          </AuthorName>
+          <AuthorAvatar>
+            <FaUserCircle />
+          </AuthorAvatar>
+        </AuthorInfoContainer>
+      </BoardInfoContainer>
```

Comment:
No dużo lepiej mi się anlizuje taki kod, gdy widzę, jak to wygląda. 
+1 do wrzucenia screena do PR.